test/statsCases/performance-error/index.js

```diff
@@ -0,0 +1,4 @@
+require("./a");
+require(["./b"]);
+require(["./c"]);
+require("lodash");
```

Comment:
Ok... but I wouldn't do that. I think this is pretty fragile because npm could decide to change that.